packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1566,6 +1616,36 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.services.TextInputConfiguration.enableIMEPersonalizedLearning}
   final bool enableIMEPersonalizedLearning;
 
+  /// {@template flutter.widgets.editableText.contentCommitMimeTypes}
+  /// Used when a user inserts image-based content through the device keyboard,
+  /// currently only used on Android.
+  ///
+  /// The passed list of strings will determine which MIME types are allowed to
+  /// be inserted via the device keyboard. If no `onContentCommitted` callback
+  /// is provided, this field will be ignored.
+  ///
+  /// {@tool snippet}
+  /// The default mime types are listed below. These are all the mime types
+  /// that are able to be handled and inserted from keyboards.
+  ///
+  /// ```dart
+  ///   List<String> defaultMimeTypes = <String>['image/png', 'image/bmp', 'image/jpg', 'image/tiff', 'image/gif', 'image/jpeg', 'image/webp'];
```

Comment:
Nit: Now that kDefaultContentCommitMimeTypes is public, maybe reference it here instead of writing out its contents.